### DISCUSSION TITLE
Sign in with GitHub to use Null Extension Description (1)

### DISCUSSION THREAD

**Discussion by TechSolomon** [community/community#15347] [Copilot Conversations] [answered]
```
Version: 1.67.0-insider
Commit: bd95a8d4516d590256eefecfce465a2f10e8f5e5
Date: 2022-04-21T05:16:16.439Z
Electron: 17.4.0
Chromium: 98.0.4758.141
Node.js: 16.13.0
V8: 9.8.177.13-electron.0
OS: Darwin x64 20.6.0
```

-----

<img width="246" alt="Screen Shot 2022-04-21 at 10 38 21 AM" src="https://user-images.githubusercontent.com/7608183/164530775-6a03159e-1eb5-4267-a558-c2ee8b6abb8f.png">
<img width="530" alt="Screen Shot 2022-04-21 at 10 36 04 AM" src="https://user-images.githubusercontent.com/7608183/164530477-78b5421c-3c4c-4f0c-9ec9-c433341d0856.png">
<img width="530" alt="Screen Shot 2022-04-21 at 10 39 49 AM" src="https://user-images.githubusercontent.com/7608183/164531022-8f965549-2e39-46cd-a73c-d844c1cb5aad.png">

**Comment by ryh** [comment, marked as answer]
I had the same problem, done with using **GitHub Copilot Nightly** version
https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-nightly
